src/main/java/in/projecteka/gateway/consent/ConsentRequestController.java

```diff
@@ -28,6 +31,8 @@
     @ResponseStatus(HttpStatus.ACCEPTED)
     @PostMapping("/v1/consent-requests/on-init")
     public Mono<Void> onDiscoverCareContext(HttpEntity<String> requestEntity) {
+        Mono<Void> toBeFiredAndForgotten = consentResponseOrchestrator.processResponse(requestEntity, X_HIU_ID);
+        toBeFiredAndForgotten.subscribe();
```

Comment:
If this is the pattern everywhere, why can't do the subscribe() inside the call itself?

Reply:
Made changes for callback API's. As this involves fixing tests, & I'm not familiar with the other API's. I left them the way they are now.